test/suites/tuned/microshift-tuned.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Documentation       Tests for verification on MicroShift's Tuned profile
+
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Test Cases ***
+Current Profile Should Be Hashed
+    [Documentation]    We expect that microshift-tuned keeps a hash of the profile and variables.
+
+    SSHLibrary.File Should Exist    /var/lib/microshift-tuned.yaml
+
+Profile Is Already Active But Cache File Is Missing
+    [Documentation]    If profile is already active, but cache file is missing,
+    ...    we expect microshift-tuned to reactivate it, reboot, and store the hashes.
+
+    Command Should Work    rm /var/lib/microshift-tuned.yaml
```

Comment:
```suggestion
    Command Should Work    rm -f /var/lib/microshift-tuned.yaml
```

Reply:
Can you add a rationale for this change? I'm not opposing, just want to know full story

Reply:
Sometime rm is aliased in the root account as `rm -i`

```
# alias rm
alias rm='rm -i'
```